pkg/avfoundation/AVFoundationBind/AVFoundationBind.h

```diff
@@ -0,0 +1,64 @@
+//
+//  AVFoundationBind.h
+//  AVFoundationBind
+//
+//  Created by Herman, Lukas on 5/23/20.
+//  Copyright © 2020 Herman, Lukas. All rights reserved.
```

Comment:
Is it non-MIT license?

Reply:
Oops, sorry, it was just automatically generated by XCode. I'll fix that.